tests/test_client.py

```
@@ -59,6 +59,36 @@ def test_client_accepts_http_token(mock_api: MagicMock) -> None:
+    ):
+        client = ZammadClient()
+
+    assert client.insecure is True
+    assert mock_instance.session.verify is False
+
+
+@patch("mcp_zammad.client.ZammadAPI")
+def test_client_insecure_mode_from_param(mock_api: MagicMock) -> None:
+    """Test that insecure constructor flag disables TLS verification."""
+    mock_instance = mock_api.return_value
+    client = ZammadClient(url="https://test.zammad.com/api/v1", http_token="test-token", insecure=True)
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "http_token"

(S106)